### PR TITLE
[13784] Add release notes of EOL releases v1.7.3, v1.8.5, v1.9.5, v1.10.1, v2.0.3, and v2.2.1

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**
Built over #337

### CODE REVIEW

**Comment by EduPonz** [comment]
@richiprosima please test this

**Comment by JLBuenoLopez-eProsima** [comment]
CI fails because the Fast DDS branch with the same name does not have the required features. [Here](http://jenkins.eprosima.com:8080/job/fastdds_docs_manual/324/) a run of the CI against Fast DDS `master` branch.

File: docs/notes/previous_versions/v2.2.1.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Version 2.2.0 (EOL)
```
Comment (EduPonz) [inline docs/notes/previous_versions/v2.2.1.rst:1]:
```suggestion
Version 2.2.1 (EOL)
```

File: docs/notes/previous_versions/v2.1.1.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Version 2.1.0
```
Comment (EduPonz) [inline docs/notes/previous_versions/v2.1.1.rst:1]:
```suggestion
Version 2.1.1
```

**Review by EduPonz — CHANGES_REQUESTED** [review]

**Review by EduPonz — APPROVED** [review]
LGTM

**Pull request merged**